docs/conf/portland/2020/ticket-choices.rst

```diff
@@ -0,0 +1,61 @@
+:template: {{year}}/generic.html
+
+Portland 2020 ticket choices
+============================
+
+As we've :doc:`announced </blog/moving-portland-prague-2020-online>`, Write the Docs Portland 2020 will be a virtual conference.
+
+If you currently have a ticket to Write the Docs Portland, your options are:
+
+- Pre-order a ticket to the online event for $100 and get the rest of your original ticket amount refunded.
+- Pre-order a ticket to the online event for $100 and contribute the rest of your original ticket amount to the Write the Docs organization.
+- Get a full refund of your original ticket amount on **May 10th, 2020**. This option will happen automatically if you don't select either of the previous options.
+
+You can submit your choice in the form below.
+
+What we can do with your contribution
+-------------------------------------
+
+Write the Docs was founded in 2013 to build a community for documentarians. There are many communities for particular technologies, or particular areas of writing. Write the Docs is quite unique in including everyone who cares about communication, documentation, and their users. That includes programmers, tech writers, developer advocates, customer support, marketers, and anyone else who wants people to have great experiences with software.
+
+Over the years, we have grown to a community that includes:
+
+* A slack network with over 9000 members.
+* Yearly conferences on 3 continents, including years of video archives.
+* Meetups in almost 50 cities.
+* A monthly newsletter with over 5000 subscribers.
+* An extensive documentation guide.
+
+We also do incidental activities, like our recent salary survey, and our currently exploring even more ways to support the community and help our community stay connected.
+
+Write the Docs is self-sustaining, and events are our main source of income.
+**We have already paid over $25,000 in venue and vendor downpayments.**
+We hope to get some of this money back,
+but if we aren't able to get this money back or additional revenue this year it would put the organization at risk.
+This is especially true if we aren't able to hold conferences again *next year*.
+
+We have many other operational expenses which our conferences pay for including:
+
+* Contractors who work on our events
+* Taxes & Bookeeping
+* Domain registration
+* Mailing lists
+* Email & help desk tools
+* Job Board hosting
+* Password management 
+* Photo hosting
+* Many other incremental expenses
```

Comment:
Not sure what "incremental" means here? Maybe we can say something like "Other administrative and marketing expenses"